src/main.ts

```diff
@@ -37,16 +39,49 @@ export class Minifier {
 
   // Recursively visits every child node, emitting text of the sourcefile that is not a part of
   // a child node.
-  visit(node: ts.Node) {
+  visit(node: ts.Node, typeChecker?: ts.TypeChecker) {
```

Comment:
Added typeChecker field on `Minifier` and added `setTypeChecker()` method on `Minifier` class.